README.md

```diff
@@ -183,8 +183,8 @@ let searchStrings = textField.rac_textSignal()
     .throttle(0.5, onScheduler: QueueScheduler.mainQueueScheduler)
 ```
 
-This prevents values from being sent less than 0.5 seconds apart, so the user
-must stop editing for at least that long before we’ll use their search string.
+This prevents values from being sent less than 0.5 seconds apart, so the user 
+must stop editing for at least that long  before we'll use their string.
```

Comment:
> so the user must stop editing for at least that long  before we'll use their string.

That part is not true. The user doesn't need to stop editing. Can you take it out?